drivers/watchdog/Kconfig.it8xxx2

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config WDT_ITE_IT8XXX2
+	bool "ITE it8xxx2 Watchdog Timer (WDT) driver"
+	depends on (SOC_IT8XXX2)
+	default y
+	help
+	  Enable WDT driver for ITE it8xxx2.
+
+config PLATFORM_EC_WATCHDOG_WARNING_LEADING_TIME_MS
```

Comment:
I suggest changing this config name to WDT_ITE_WARNING_LEADING_TIME_MS.

Reply:
Done